warehouse/core/src/main/java/datawave/ingest/protobuf/Uid.java

```diff
@@ -135,7 +135,7 @@ public List getDefaultInstanceForType() {
             return defaultInstance;
         }
 
-        private final com.google.protobuf.UnknownFieldSet unknownFields;
+        private final transient com.google.protobuf.UnknownFieldSet unknownFields;
```

Comment:
Same for this, won't this change get reverted when re-generated?

Reply:
reverted.